janus-idp/operator
# Bump Ginkgo to v2.16.0

## Description
This bumps Ginkgo from v2.9.5 to v2.16.0.
I was running some tests with the Gingko CLI and a programmatic focus. Then I noticed that CLI filters would always override any programmatic focus, which can be confusing.
This issue was fixed in Ginkgo v2.11.0. See https://github.com/onsi/ginkgo/blob/master/CHANGELOG.md#2110

This PR bumps it to the latest version available at this time.

## Which issue(s) does this PR fix or relate to

&mdash;

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
1. Make sure to remove any existing ginkgo binary under your `bin` folder, so that the expected version is downloaded locally:
```sh
$ rm -f bin/ginkgo
```
2. `make test` and `make test-e2e` should pass.
